packages/messaging-api-line/src/LineTypes.ts

```diff
@@ -848,6 +848,17 @@ export type BasicAudienceGroup = {
 
   // When the audience was created (in UNIX time).
   created: number;
+
+  // The value specified when creating an audience for uploading user IDs to indicate the type of accounts that will be given as recipients. One of:
+  // - true: Accounts are specified with IFAs.
+  // - false (default): Accounts are specified with user IDs.
+  isIfaAudience: string;
+
+  // Audience's update permission. Audiences linked to the same channel will be
```

Comment:
this comment seems to be cut in the middle

Reply:
Thanks

Reply:
Thanks.